src/main/java/org/demo/controller/CxboxRestController.java

```diff
@@ -55,6 +55,7 @@ public enum CxboxRestController implements EnumBcIdentifier {
 		contactEditAssoc(clientEdit, ClientContactService.class),
 		companySuggestionPickList(clientEdit, CompanyService.class),
 	meeting(MeetingReadService.class),
+	meetingClientList(clientEdit, MeetingReadService.class),
```

Comment:
move it under parent (clientEdit bc) and use indentatation to show it is child bc